README.md

```diff
@@ -26,7 +26,7 @@ Run the latest production build:
             -e MYSQL_USER=kimai \
             -e MYSQL_PASSWORD=kimai \
             -e MYSQL_ROOT_PASSWORD=kimai \
-            -p 3399:3306 -d mysql
+            -p 3306:3306 -d mysql
```

Comment:
This was a deliberate setting. If a user has WAMP installed, or a native mysql DB then we get a port conflict. I spent some time when first deploying this container repeating asking users to shut down their local mysql and try again.

```suggestion
            -p 3399:3306 -d mysql
```

Reply:
Probably when changing back the port to 3399 there should be a sentence added about that fact and how to reach the database locally.